Netology_IB_Instruments/Views/Profile/ProfileViewController.swift

```diff
@@ -36,31 +36,43 @@ class ProfileViewController: UIViewController {
         return table
     }()
 
-//    private lazy var blackView: UIView = {
-//        let view = UIView()
-//        view.backgroundColor = .black
-//        view.alpha = 0
-////        view.addSubview(blackViewProfileImage)
-//        return view
-//    }()
-
-//    private lazy var blackViewProfileImage: UIImageView = {
-//        let image = UIImageView(image: UIImage(named: "wolk"))
-//        image.layer.masksToBounds = true
-//        image.layer.cornerRadius = 64
-//        image.layer.borderColor = UIColor.white.cgColor
-//        image.backgroundColor = .black
-//        image.layer.borderWidth = 3
-//        image.frame = CGRect(x: 0, y: 0, width: 100, height: 100)
-//        return image
-//    }()
+    private lazy var blackView: UIView = {
+        let view = UIView()
+        view.backgroundColor = UIColor(displayP3Red: 0, green: 0, blue: 0, alpha: 0.7)
+        view.alpha = 0
+        [blackViewProfileImage, blackViewCross].forEach{
+            view.addSubview($0)
+        }
+        return view
+    }()
+
+    private lazy var blackViewProfileImage: UIImageView = {
+        let image = UIImageView(image: UIImage(named: "wolk"))
```

Comment:
Хорошо бы, чтобы это изображение использовало ту же картинку, что и profileHeader. Иначе может получиться так, что пользователь поменяет аватар, а при ближайшем рассмотрении увидеть wolk'а

Reply:
Хорошо. Можно поступить так:
```suggestion
        let image = UIImageView(image: profileHeader.profileImage.image)
```
В этом моменте немного поленился и особо не думал. Спасибо, что обратили внимание на эту ошибку. 
Я был бы очень благодарен, если бы преподаватели при проверке моих работ судили их максимально строго. Уверен, это далеко не единственная моя ошибка.